syz-manager/manager.go

```diff
@@ -745,6 +746,9 @@ func (mgr *Manager) Check(a *CheckArgs, r *int) error {
 	if mgr.cfg.Cover && !a.Kcov {
 		Fatalf("/sys/kernel/debug/kcov is missing. Enable CONFIG_KCOV and mount debugfs")
 	}
+	if mgr.cfg.Sandbox == "namespace" && !a.UserNamespaces {
+		return errors.New("requested namespace sandbox but user namepsaces are not enabled")
```

Comment:
You need to call `Fatalf` as above. Returning an error from rpc function will just fail the call on client side, which produce an obscure VM crash and no indication as to what's the root cause.